inventory/rh-sso/group_vars/rh-sso-hosts.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+ansible_user: cloud-user
+ansible_become: True
+
+list_of_packages_to_install:
+  - "@rh-sso7"
+  - "firewalld"
+
+add_user_keycloak: "/opt/rh/rh-sso7/root/usr/share/keycloak/bin/add-user-keycloak.sh"
```

Comment:
This doesn't seem like it should be part of the inventory. If you believe the user will need to override this, maybe just add it to the `defaults/main.yml` of the role to allow for an override if needed.

Reply:
Sure thing. This is a pre-defined value and it will probably not need to be overridden.
Fixed.